### PR TITLE
doc: add `request-validation` plugin chinese document.

### LINKED ISSUE #1882: doc: Chinese doc for plugin request-validator
here is the English doc: https://github.com/apache/incubator-apisix/commit/a6179996d4ddd8f99dfc44398666c282e6fd78ba#diff-14449a90466a7b8b69021054e24a789b

### PR DESCRIPTION
**Pull request by shuaijinchao**
### What this PR does / why we need it:
FIX #1882 

### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [x] Have you modified the corresponding document?
* [x] Is this PR backward compatible?

### CODE REVIEW

**Comment by membphis** [comment]
minor style, please take a look at the output of CI. https://github.com/apache/incubator-apisix/pull/1932/checks?check_run_id=924153732#step:6:60

**Comment by shuaijinchao** [comment]
> minor style, please take a look at the output of CI. https://github.com/apache/incubator-apisix/pull/1932/checks?check_run_id=924153732#step:6:60

fixed, many thx.

File: doc/zh-cn/plugins/request-validation.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+            default = "enum_string_1"
+        }
+    }
+}
+```
+
+**验证 多级`Json`:**
+
+```shell
+"body_schema": {
```
Comment (membphis) [inline doc/zh-cn/plugins/request-validation.md:128]: that is wrong, please take a look at the English version

Comment (shuaijinchao) [reply doc/zh-cn/plugins/request-validation.md:128]: fixed, please recheck.

Comment (membphis) [reply doc/zh-cn/plugins/request-validation.md:128]: https://github.com/apache/incubator-apisix/blob/master/doc/plugins/request-validation.md#examples

that is the right version, your example is wrong.

Comment (shuaijinchao) [reply doc/zh-cn/plugins/request-validation.md]: OK, the json format problem has been fixed.

**Comment by membphis** [comment]
@shuaijinchao merged, many thx

**Pull request merged**